structure/0062-quilt-community-collab.md

```diff
@@ -0,0 +1,201 @@
+# Summary
+
+This document intends to describe the Quilt Community Collab Program, as well as its entry requirements, processes, and the roles that may be assigned to its members.
+
+# Motivation
+
+The Minecraft community is complex and diverse and, in order to keep everyone safe, it's often useful for communities to share information with each other. The main goal of the Quilt Community Collab Program is to facilitate this exchange of information between a selected group of communities, which have been vetted for trustworthiness and educated on the best way to approach this type of data.
+
+The Minecraft community at large has long been host to a surprisingly high amount of hatred and discrimination, and the Quilt Community Collab Program is one of our ways of giving back to the community that got us where we are --- with the aim of hopefully improving it for everyone.
+
+# Explanation
+
+The Quilt Community Collab Program (henceforth, "the Program") largely exists as a private Discord server, wherein representatives from participating Minecraft-related communities exchange information and coordinate on issues of moderation, alongside Quilt's Community Team and individual contributors that don't represent specific communities.
+
+It also exists to serve as a coordination point for events and other cross-community projects within the Minecraft sphere.
+
+## What the Program is not
+
+* An official, direct affiliation with the Quilt project, or any other community that participates in the Program
+* A space for building or coordinating the character assassinations of community members or prominent personalities
+* A space meant for blindly synchronizing punishments between communities
+  * Instead, each community must decide on appropriate moderation actions for their own spaces, which may be informed by the data they're given access to, but not blindly copied between communities --- Breaking this rule is likely to result in a community being removed from the Program
+
+## Program Roles
+
+As the Program exists largely on Discord, the roles of its members are directly represented by Discord server roles.
```

Comment:
This looks to not resolve @gdude2002's comment about the inclusion of the Quilt Administrative Board as oversight on the Quilt Community Team (https://github.com/QuiltMC/rfcs/pull/65#issuecomment-1272516842), unless that is covered by the section on Project Roles or I'm otherwise mistaken.

(I'd argue that, since the Quilt Administrative Board does not have moderation duties in the community spaces, it would be inappropriate to give them the Collaborator role.)

Reply:
Quilt admins shouldn't have the collaborator role. I'm not sure what to do with them right now, though.

Reply:
I thought we agreed on adding community admins to the collaborator role?